integration/querydsl/expressions/src/main/java/com/blazebit/persistence/querydsl/JPQLNextExpressions.java

```diff
@@ -1122,7 +1123,7 @@ public static <T> Expression<T> cast(Class<T> result, Expression<?> expression)
      * @param <T> Expression type
      * @return The treated expression
      */
-    public static <T> Expression<T> treat(Class<T> result, Expression<?> expression) {
+    public static <T> SimpleOperation<T> treat(Class<T> result, Expression<?> expression) {
```

Comment:
Is this change necessary? I couldn't find uses of the method in the changed code, so maybe this turns out to be just a cosmetic change as well? If you want, we can change the signatures in 2.0.

Reply:
Not necessary no. I'll revert it.

Reply:
The change was made to use in JPQLNextSerializer which initially replaced the mod operation expression with the expression returned from `treat(operator.getType(), operator)`. 

However this has since changed to simply `"TREAT_" + type.getSimpleName().toUpperCase()` because I didn't like the code.

This was a left over from that change and I have reverted it now.